src/physics/em/detail/MollerBhabhaInteractor.i.hh

```diff
@@ -22,23 +22,26 @@ namespace detail
 /*!
  * Construct with shared and state data.
  *
- * The incident particle must be above the energy threshold: this should be
- * handled in code *before* the interactor is constructed.
+ * The incident particle must be within the model's valid energy range. this
+ * must be handled in code *before* the interactor is constructed.
  */
 CELER_FUNCTION MollerBhabhaInteractor::MollerBhabhaInteractor(
     const MollerBhabhaPointers& shared,
     const ParticleTrackView&    particle,
+    const CutoffView&           cutoffs,
     const Real3&                inc_direction,
     StackAllocator<Secondary>&  allocate)
     : shared_(shared)
     , inc_energy_(particle.energy().value())
     , inc_momentum_(particle.momentum().value())
     , inc_direction_(inc_direction)
+    , secondary_energy_cutoff_(cutoffs.energy().value())
     , allocate_(allocate)
     , inc_particle_is_electron_(particle.particle_id() == shared_.electron_id)
 {
     CELER_EXPECT(particle.particle_id() == shared_.electron_id
                  || particle.particle_id() == shared_.positron_id);
+    CELER_EXPECT(cutoffs.energy().value() < inc_energy_);
```

Comment:
Since the min sampled energy depends only on the initialization data (incident energy, cutoffs) I'd move your `min_sampled_energy` below into this function:
```suggestion
    CELER_EXPECT(inc_energy_ > secondary_energy_cutoff_);
    secondary_energy_cutoff_ = celeritas::max(
        secondary_energy_cutoff_,
        (inc_particle_is_electron_ ? 2 : 1) * shared_.min_valid_energy());
```
(since the model's lower energy is particle-dependent)